apps/dashboard/app/javascript/packs/files/DataTable.js

```diff
@@ -0,0 +1,427 @@
+import 'datatables.net';
+import 'datatables.net-bs4/js/dataTables.bootstrap4';
+import 'datatables.net-select';
+import 'datatables.net-select-bs4';
+import Handlebars from 'handlebars';
+export {CONTENTID, TRIGGERID};
+
+const TRIGGERID = {
+	changeDirectory: 'changeDirectory',
+    changeDirectoryPrompt: 'changeDirectoryPrompt',	
+	clearClipboard: 'clearClipboard',
+	closeSwal: 'closeSwal',
+    copyFile: 'copyFile',
```

Comment:
Be careful of formatting here. Plus you have event-names like `clearClipboard` here even though it doesn't fire or listen for this event. Is this the right place for it?

Reply:
hrmmm on the formatting, i'll have to check and see what happened there.  

As for the clearClipboard event name.  You ask a good question.  I actually asked myself that same question when working on this.  I had a couple options for defining the event names.  I could have 1 hash of event names and use it everywhere, this way I am ever exporting/importing 1 hash.  The other option would be have a hash for each class and then export/import multiple hashes.  Either way there is waste.  My thoughts are it'd be better to have a single hash to contain the event names and pass it around instead of having multiple hashes and passing multiple hashes around.